### PR TITLE
VS-1029 Fix naming errors in hail_gvs_import script

### PR DESCRIPTION
**Pull request by gbggrant**
Fix naming errors in script and added the ability to specify the temp directory for Hail.

### CODE REVIEW

File: scripts/variantstore/wdl/extract/hail_create_vat_inputs.py [hunk 1]
```diff
@@ -184,11 +185,17 @@ def write_tie_out_vcf(vds, vcf_output_path):
...
+                        default="@SITES_ONLY_VCF_OUTPUT_PATH@"),
+    parser.add_argument('--temp_path', type=str, help='Path to temporary directory', default="@TEMP_DIR@",
+                        required=True)
 
     args = parser.parse_args()
 
-    vds = hl.vds.read_vds(args.vds_path)
-    local_ancestry_file = create_vat_inputs.download_ancestry_file(args.ancestry_file)
+    temp_path = args.temp_path if not args.temp_path.endswith('/') else args.temp_path[:-1]
+    time_stamp = datetime.today().strftime('%Y-%m-%d_%H:%M:%S')
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/hail_create_vat_inputs.py:195]: it is [strongly recommended](https://cloud.google.com/storage/docs/objects#naming) not to use colons in GCS object names (yeah for Windows reasons, but still)

File: scripts/variantstore/wdl/extract/hail_create_vat_inputs.py [hunk 2]
```diff
@@ -184,11 +185,17 @@ def write_tie_out_vcf(vds, vcf_output_path):
     parser.add_argument('--ancestry_input_path', type=str, help='Input ancestry file path', required=True)
     parser.add_argument('--vds_input_path', type=str, help='Input VDS path', default="@VDS_INPUT_PATH@")
     parser.add_argument('--sites_only_output_path', type=str, help='Output sites-only VCF path',
-                        default="@SITES_ONLY_VCF_OUTPUT_PATH@")
+                        default="@SITES_ONLY_VCF_OUTPUT_PATH@"),
+    parser.add_argument('--temp_path', type=str, help='Path to temporary directory', default="@TEMP_DIR@",
+                        required=True)
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/hail_create_vat_inputs.py:190]: having a combination of `required=True` and a `default` seems contradictory

**Review by RoriCremer — APPROVED** [review]
thank you for adding the date!

File: scripts/variantstore/wdl/extract/hail_create_vat_inputs.py (hunk 1 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/extract/hail_create_vat_inputs.py]: Thanks. It's a darn shame though.

File: scripts/variantstore/wdl/extract/hail_create_vat_inputs.py (hunk 2 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/extract/hail_create_vat_inputs.py]: Thanks, resolved.

**Comment by gbggrant** [comment]
@mcovarr I've plumbed cloud_sdk_slim_docker through.

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl [hunk 3]
```diff
@@ -47,7 +49,7 @@ workflow GvsCreateVATFilesFromBigQuery {
             contig_array = contig_array,
             output_path = output_path,
             merge_vcfs_disk_size_override = merge_vcfs_disk_size_override,
-            cloud_sdk_docker = effective_cloud_sdk_docker,
+            cloud_sdk_docker = effective_cloud_sdk_slim_docker,
```
Comment (mcovarr) [inline scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl:52]: input name should also be `slim`

**Review by mcovarr — CHANGES_REQUESTED** [review]
`GvsCreateVATFilesFromBigQuery` should also get the slim docker and pass it as a parameter to `GvsCreateVATFilesFromBigQuery` when it calls it.

File: scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl (hunk 3 above)
Comment (gbggrant) [reply scripts/variantstore/variant_annotations_table/GvsCreateVATFilesFromBigQuery.wdl]: Okay - resolved this and the parameterization of `GvsCreateVATFromVDS.wdl`

**Comment by gbggrant** [comment]
@mcovarr when you have a moment can you recheck this.

**Review by mcovarr — APPROVED** [review]
LGTM!

**Pull request merged**